src/components/auth/profile/CertificatesTab.tsx

```diff
@@ -1,11 +1,157 @@
+          flexGrow: 1,
+          padding: '10px 30px',
+        }}>
+        <Box>
+          <h3 className={classes.thinFont}>Онлайн дарения</h3>
+        </Box>
+        <Box
+          sx={{
+            display: 'flex',
+            alignItems: 'baseline',
+            justifyContent: 'space-between',
+          }}>
+          <span className={classes.smallText}>Покажи:</span>
+          <Box>
+            <Checkbox defaultChecked />
+            <span className={classes.smallText}>еднократни</span>
+          </Box>
+          <Box>
+            <Checkbox defaultChecked />
+            <span className={classes.smallText}>месечни</span>
+          </Box>
+          <LocalizationProvider dateAdapter={AdapterDateFns}>
+            <span className={classes.smallText}>от дата</span>
+            <DesktopDatePicker
+              label="от дата"
+              inputFormat="MM/dd/yyyy"
+              value={fromDate}
+              onChange={(date) => setFromDate(date as Date)}
+              renderInput={(params) => <TextField {...params} />}
+            />
+            <span className={classes.smallText}>до дата</span>
+            <DesktopDatePicker
+              label="до дата"
+              inputFormat="MM/dd/yyyy"
+              value={toDate}
+              onChange={(date) => setToDate(date as Date)}
+              renderInput={(params) => <TextField {...params} />}
+            />
+          </LocalizationProvider>
+        </Box>
+        {data.donations.length ? (
+          <TableContainer>
+            <Table sx={{ minWidth: 650, backgroundColor: 'white' }} aria-label="simple table">
+              <TableHead>
+                <TableRow>
+                  <TableCell>№</TableCell>
+                  <TableCell>Дата</TableCell>
+                  <TableCell>Вид</TableCell>
+                  <TableCell>Кауза</TableCell>
+                  <TableCell>стойност</TableCell>
+                  <TableCell>сертификат</TableCell>
+                </TableRow>
+              </TableHead>
+              <TableBody>
+                {data.donations.map((donation, index) => (
+                  <TableRow key={index} sx={{ '&:last-child td, &:last-child th': { border: 0 } }}>
+                    <TableCell component="th" scope="row">
+                      {index + 1}
+                    </TableCell>
+                    <TableCell>{formatDateString(donation.createdAt)}</TableCell>
+                    <TableCell>
+                      <Avatar sx={{ background: '#F6992B' }}>
+                        <StarIcon></StarIcon>
+                      </Avatar>
+                    </TableCell>
+                    <TableCell>{donation.targetVault.campaign.title}</TableCell>
+                    <TableCell>
+                      {donation.amount} {donation.currency}
+                    </TableCell>
+                    <TableCell>
+                      <Button variant="outlined">
+                        Свали <ArrowForwardIcon></ArrowForwardIcon>
```

Comment:
Every component without children can and it's preferred to be used with self-closing syntax

 ```suggestion
                        Свали <ArrowForwardIcon />
```